<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReviewDeleteDialog</class>
 <widget class="QDialog" name="ReviewDeleteDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ReviewDeleteDialog</string>
  </property>
  <widget class="QPushButton" name="pbOk">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>140</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCancel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>140</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbSelected">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>200</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>Selected Samples</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbAll">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>200</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>All Samples</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
